api/apps/geoprocessing/src/modules/planning-area/planning-area.controller.ts

```diff
@@ -25,10 +24,8 @@ export class PlanningAreaController {
     @Body() shapefileInfo: Express.Multer.File,
   ): Promise<PlanningAreaDto> {
     try {
-      return plainToClass<PlanningAreaDto, PlanningAreaDto>(
-        PlanningAreaDto,
-        await this.planningAreaService.save(shapefileInfo),
-      );
+      const result = await this.planningAreaService.save(shapefileInfo);
+      return PlanningAreaDto.from(result);
```

Comment:
It isn't that I don't like it but why not `serializer` as we commonly use it around? Just because of unnecessary boilerplate?

Reply:
do you mean `serialize` and `serializerConfig` from the base service?

yes,
it's not a regular entity, we do not keep the GeoJSON (and pu sizes) in DB, at least not in this shape
also, there is always only one entity returned at a time

Reply:
added to be consistent